webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -103,14 +104,37 @@ function GithubItems(props) {
             }
         }
 
+        let hasConflict = '';
+        if (item.mergeable != null && !item.mergeable) {
+            hasConflict = (
+                <OverlayTrigger
+                    key='githubRHSPRMergeableIndicator'
+                    placement='top'
+                    overlay={
+                        <Tooltip id='githubRHSPRMergeableTooltip'>
+                            {'This pull request has conflicts that must be resolved'}
+                        </Tooltip>
+                    }
+                >
+                    <span>
+                        <Octicon
```

Comment:
(0/5) The way this icon is used for conflicts, the color is not pure black, but (according to my browser) #6a737d. Maybe using that color would look better.

![Screenshot from 2020-05-29 10-36-33](https://user-images.githubusercontent.com/1933730/83239527-51efea80-a198-11ea-9aa2-752e99fc5f58.png)

Reply:
Also the size looks a bit off to my eye, but I am not 100%. Maybe involve UX?

Reply:
@asaadmahmood :point_up_2:

Reply:
Agreed on size and possibly also spacing. Could you update the screenshot once updated with the mattermost icon as suggested by Asaad?